### PR TITLE
feat(thirdparty): Introduce abseil

### PR DESCRIPTION
**Pull request by Samunroyu**
### What problem does this PR solve? 
Introduce abseil as thridparty for using abseil_strings method  

### What is changed and how does it work?
Abseil LTS 20230802.1 is introduced to thridparty/CMakelist.txt and to complie.

##### Related changes
kms key manager client need to use Abseil_strings some methods.

### CODE REVIEW

File: thirdparty/CMakeLists.txt [hunk 1]
```diff
@@ -409,3 +409,9 @@ ExternalProject_Add(http-parser
         INSTALL_COMMAND cp -R http_parser.h ${TP_OUTPUT}/include/nodejs/http_parser.h
         BUILD_IN_SOURCE 1
         )
+
+ExternalProject_Add(abseil
+        URL https://github.com/abseil/abseil-cpp/archive/refs/tags/20230802.1.zip
+        URL_MD5 5c6193dbc82834f8e762c6a28c9cc615
+        -DABSL_FIND_GOOGLETEST=OFF
```
Comment (acelyc111) [inline thirdparty/CMakeLists.txt:416]:
```suggestion
        CMAKE_ARGS -DCMAKE_INSTALL_PREFIX=${TP_OUTPUT}
        -DABSL_FIND_GOOGLETEST=OFF
```

**Review by acelyc111 — DISMISSED** [review]

File: thirdparty/CMakeLists.txt [hunk 2]
```diff
@@ -409,3 +409,10 @@ ExternalProject_Add(http-parser
         INSTALL_COMMAND cp -R http_parser.h ${TP_OUTPUT}/include/nodejs/http_parser.h
         BUILD_IN_SOURCE 1
         )
+
+ExternalProject_Add(abseil
+        URL https://github.com/abseil/abseil-cpp/archive/refs/tags/20230802.1.zip
+        URL_MD5 5c6193dbc82834f8e762c6a28c9cc615
+        CMAKE_ARGS -DCMAKE_INSTALL_PREFIX=${TP_OUTPUT}
+	-DABSL_FIND_GOOGLETEST=OFF
```
Comment (acelyc111) [inline thirdparty/CMakeLists.txt:417]: Use spaces instead of tabs.

**Review by acelyc111 — APPROVED** [review]

**Review by acelyc111 — APPROVED** [review]

**Review by empiredan — APPROVED** [review]

File: thirdparty/CMakeLists.txt [hunk 3]
```diff
@@ -409,3 +409,11 @@ ExternalProject_Add(http-parser
         INSTALL_COMMAND cp -R http_parser.h ${TP_OUTPUT}/include/nodejs/http_parser.h
         BUILD_IN_SOURCE 1
         )
+
+ExternalProject_Add(abseil
+        URL https://github.com/abseil/abseil-cpp/archive/refs/tags/20230802.1.zip
```
Comment (GehaFearless) [inline thirdparty/CMakeLists.txt:414]: could U add one more url?

`https://pegasus-thirdparty-package.oss-cn-beijing.aliyuncs.com/abseil-20230802.1.zip.zip`

Comment (GehaFearless) [reply thirdparty/CMakeLists.txt:414]: Be like ` ${OSS_URL_PREFIX}/abseil-20230802.1.zip`

**Review by acelyc111 — APPROVED** [review]

**Review by GehaFearless — APPROVED** [review]

**Pull request merged**